lib/OpenQA/Worker/Job.pm

```diff
@@ -544,23 +513,18 @@ sub _format_reason {
     my ($self, $result, $reason) = @_;
 
     # format stop reasons from the worker itself
-    if ($reason eq 'setup failure') {
-        return "setup failure: $self->{_setup_error}";
-    }
-    elsif ($reason eq 'api-failure') {
+    return "setup failure: $self->{_setup_error} if $reason eq 'setup failure';
```

Comment:
I prefer the previous style because it looks more uniform. I don't think the condensed form is necessarily always easier to read.

Reply:
reverted